packages/front-end/components/Settings/Team/MemberList.tsx

```diff
@@ -149,6 +150,9 @@ const MemberList: FC<{
                     <td>
                       {member.dateCreated && datetime(member.dateCreated)}
                     </td>
+                    <td>
+                      {member.lastLoginDate && date(member.lastLoginDate)}
```

Comment:
I would consider rounding it to the day so as to avoid confusing the user. You could use `data-fns` to return the corresponding day in the user locale too.

Reply:
Ha sorry I read too fast.